packages/wp-now/src/execute-wp-cli.ts

```diff
@@ -0,0 +1,41 @@
+import startWPNow from './wp-now';
+import { downloadWPCLI } from './download';
+import { disableOutput } from './output';
+import getWpCliPath from './get-wp-cli-path';
+import getWpNowConfig from './config';
+import { DEFAULT_PHP_VERSION, DEFAULT_WORDPRESS_VERSION } from './constants';
+
+/**
+ * This is an unstable API. Multiple wp-cli commands may not work due to a current limitation on php-wasm and pthreads.
+ * @param args The arguments to pass to wp-cli.
+ */
+export async function executeWPCli(args: string[]) {
+	await downloadWPCLI();
+	disableOutput();
+	const options = await getWpNowConfig({
+		php: DEFAULT_PHP_VERSION,
+		wp: DEFAULT_WORDPRESS_VERSION,
+		path: process.env.WP_NOW_PROJECT_PATH || process.cwd(),
+	});
+	const { phpInstances, options: wpNowOptions } = await startWPNow({
+		...options,
+		numberOfPhpInstances: 2,
+	});
+	const [, php] = phpInstances;
+
+	try {
+		php.useHostFilesystem();
+		await php.cli([
+			'php',
+			getWpCliPath(),
+			`--path=${wpNowOptions.documentRoot}`,
+			...args,
+		]);
+	} catch (resultOrError) {
+		const success =
```

Comment:
FYI I merged this change: https://github.com/WordPress/wordpress-playground/pull/470/files#diff-3652c54df4b8d0d8e367481121e32f08cc409cfc4236fdb3172d69dcbf6ca8b0

It shouldn't matter here at all and this check still passes, but I wanted to flag it here just in case.